openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetailPage.tsx

```diff
@@ -253,7 +254,11 @@ const PoliciesDetailPage = () => {
                       onClick={(e) => {
                         e.stopPropagation();
                         history.push(
-                          getEditPolicyRulePath(fqn, rule.name || '')
+                          getEditPolicyRulePath(
+                            fqn,
+                            // rule name is same as entity fqn so we need to encode it to pass it as a param
+                            getEncodedFqn(rule.name || '')
+                          )
```

Comment:
@Sachin-chaurasiya  we can encode this too in the `getEditPolicyRulePath` just like we are doing for the `FQN` ?
<img width="550" alt="image" src="https://github.com/user-attachments/assets/7f2efc4e-7932-4165-9356-0a0ff2e87c17">